docs/product-release-notes/src/main/asciidoc/con-bxms-rn-whats-new.adoc

```diff
@@ -0,0 +1,152 @@
+| processInstanceID
+| customProcessID
+
+.5+| Activity that has failed
+| activityID
+| activityName
+| activityType
+| iteration
+| status
+
+.2+| Error data
+| errorMessage: Message with the error that has occurred.
+| errorStack: Stack with error detail.
+|===
+--
+
+The process instance list includes a new column that displays the number of errors per process instance which are not marked as acknowledged. When you select the column, a window appears which will enables you to navigate to the new Error perspective.
+
+The Execution Errors perspective lists any errors that occur during process instances or executor service jobs on a Decision Server. Use this perspective to help troubleshoot issues. It is visible to users with the admin and process-admin roles.
+
+
+== Process Engine
+
+=== Process and task administration API
+A process administration API and task administration API have been introduced to simplify some of the more complex administrator use cases.
+The process admin API enables you to:
+
+* Retrieve all process definition nodes
+* Cancel node instances
+* Retrigger node instances
+* Update the timer (absolute or relative)
+* List timer instances
+* Trigger nodes
+
+The task administration API enables you to:
+
+* Add and remove potential owners and excluded owners and business administrators
+* Add and remove task inputs and outputs
+* List, create, and cancel escalations and notifications
+
+=== Advanced task routing
+When tasks are assigned to a group of users, you can use pluggable task assignment strategies to automatically assign tasks to a suitable individual immediately. This enables more efficient task allocation, based on all properties associated with the task (for example potential owners and task priority but also task data that includes information such as geography, required skills, and so forth). You can use business rules to define the assignment logic, making it easy to customize this to your needs.
+
+=== Process execution server
+The process execution server (also known as kie-server) has been extended to support core engine features and offers a remote API for these operations. In addition the following architectural changes were introduced.
+
+==== Separate Business Central from execution server
+Business Central now delegates all of its requests to the execution server as well. The main advantage is that Business Central can now be used to monitor any set of execution servers. By linking the execution server to Business Central, the process and task monitoring UIs in Business Central can now connect to this execution server and show all relevant information. When multiple independent execution servers are used, you can either connect to a specific one or use the smart router to aggregate information across multiple servers. 
+
+==== Smart router
+The smart router (also known as kie-server-router) can be used as a proxy to help manage multiple independent process execution servers.
+
+=== Aliases
```

Comment:
This also relates to execution server, maybe add it under there as well using ====?

Reply:
Kris, would you please clarify. Are you referring to Aliases? And can you please tell me where you think I should add it? (Sorry, still finding my way around the product).

Reply:
@emmurphy1 I think this would fit nice under "Process execution server" section (line 116), so next to "Separate Business Central from execution server" and "Smart Router".  So if you just add another '=' to Aliases (so it becomes "====") it will show up as a third item under "Process execution server" section.

Reply:
Ah, I see! Thank you.